morebits.js

```diff
@@ -1737,7 +1739,7 @@ Morebits.wiki.page = function(pageName, currentAction) {
 		botEdit: false,
 		pageSection: null,
 		maxConflictRetries: 2,
-		maxRetries: 2,
+		maxRetries: 3,	// temporary increase from 2 to make batchundelete more likely to succeed [[phab:T222402]] #613
```

Comment:
Isn't it also okay to use `wikipedia_page.setMaxConflictRetries(3)` in batchundelete.js, rather than changing the default for every object?

Reply:
`setMaxRetries`, but yes, I take your point.  Doing it this way shouldn't hammer the API elsewhere (nothing *else* is failing AFAIK) and, more importantly, is something I can do on-wiki to alleviate the issue without needing code review.

Reply:
Oh you know, nevermind; rewriting `batchundelete` to use `morebits.wiki.page.undeletePage` is what made `maxRetries` even relevant in the first place, so my point is dumb and you're right!